presto-hive-hadoop2/src/test/java/io/prestosql/plugin/hive/metastore/cache/TestCachingHiveMetastoreWithQueryRunner.java

```diff
@@ -0,0 +1,119 @@
+package io.prestosql.plugin.hive.metastore.cache;
```

Comment:
> CachingHiveMetastore didn't work correctly with GRANT/REVOKE

Instead of saying what was broken, please describe how it got improved. I would say:

```
Respect GRANT and REVOKE CachingHiveMetastore
```

Reply:
will split into 3 commits a moment before merge, doing this on every fix is clumsy

Reply:
splitted into 2 commits